diyhue-flask/config.json

```diff
@@ -25,7 +25,7 @@
 	"auth_api": true,
 	"hassio_api": true,
 	"hassio_role": "default",
-	"host_network": true,
+	"host_network": false,
```

Comment:
This is the default value, and should be removed instead.

Reply:
Thanks - fixed

Reply:
There are quite a few other things in this config that are defaults. You might want to consider using: <https://github.com/marketplace/actions/frenck-s-home-assistant-add-on-linter>

Reply:
@MaxBec is it best I leave that to you?

Reply:
Thank you guys. Will do so!